ios/FluentUI.Demo/FluentUI.Demo/Demos/NavigationControllerDemoController.swift

```diff
@@ -66,18 +74,30 @@ class NavigationControllerDemoController: DemoController {
         presentController(withLargeTitle: true, style: .system, accessoryView: createAccessoryView(with: .darkContent), contractNavigationBarOnScroll: true)
     }
 
+    @objc func showLargeTitleWithSystemStyleShyAccessoryAndSubtitle() {
+        presentController(withLargeTitle: true, subtitle: "Subtitle goes here", style: .system, accessoryView: createAccessoryView(with: .darkContent), contractNavigationBarOnScroll: true)
+    }
+
     @objc func showLargeTitleWithSystemStyleAndFixedAccessory() {
         presentController(withLargeTitle: true, style: .system, accessoryView: createAccessoryView(with: .darkContent), contractNavigationBarOnScroll: false)
     }
 
-    @objc func showRegularTitleWithShyAccessory() {
```

Comment:
maybe the better way to handle this is to do a deprecation warning to ease people to moveto the new api

Reply:
This function is internal to the FUA test app. I'm just correcting a typo. :-p